sycl/test/on-device/sub_group/load_store.cpp

```diff
@@ -1,5 +1,5 @@
 
-// UNSUPPORTED: cuda || cpu
+// XFAIL: cuda, cpu
```

Comment:
It is better to have UNSUPPORTED: on cpu
The problem is seen only for some ISAs. See https://github.com/intel/llvm/issues/2252
```suggestion
// UNSUPPORTED: cpu
// XFAIL: cuda
```